DP-64: Add meta descriptions to each page in the operation guide. (#6340)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/device-config-backup/introduction.adoc

= Device Configuration Backup

> :description: Overview of how to back up configuration for devices like routers, firewalls, and switches with OpenNMS Horizon/Meridian, for better disaster recovery.

Reviewer: Can also be utilized for "change management", if we're looking to enhance page metadata

> :description: Overview of how to back up configuration for devices like routers, firewalls, and switches with OpenNMS Horizon/Meridian, for better disaster recovery.

Author: ```suggestion
:description: Back up configuration for devices like routers, firewalls, switches with OpenNMS Horizon/Meridian, for better disaster recovery and change management.
```
